content/en/docs/Tasks/update-operator-to-next-version.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Update operator to next version"
+date: 2021-03-09
+weight: 2
+description: >
+   Given an operator at version n, what should I do to create a version n+1.
+--- 
+
+## Introduction
+
+In the Operator Lifecycle Manager (OLM) ecosystem, the following resources are used to resolve Operator installations and upgrades:
+
+1. `ClusterServiceVersion (CSV)` - A YAML manifest created from Operator metadata that assists the Operator Lifecycle Manager (OLM) in running the Operator in a cluster.
+
+2. `CatalogSource` - Operator metadata, defined in CSVs, can be stored in a collection called a CatalogSource.
```

Comment:
This is technically incorrect as CSVs aren't the only mechanism for exposing Operator metadata as you can also use a ConfigMap instead. I'm not sure if we market that approach anymore in the more recent 4.x releases.